doc/user_guide.md

```diff
@@ -47,54 +48,45 @@ The Xively Client was written with the understanding that the IoT Field might ch
-* Check to see if an OCSP response has not expired.
+* A check to see if the current date is past the Certificate's Activation Date, and that the current time precedes the Certificate's Expiration Date.
+* A check to see if an OCSP response has not expired.
     * You can read more about OCSP Stapling here: http://en.wikipedia.org/wiki/OCSP_stapling
 
-Without an accurate clock these activation / validation tests will be inaccurate. Even worse, if the time is substantially inaccurate (like a device that thinks it's 1970, for instance) then TLS handshaking will fail and the device will never connect to the Xively Service.
+Without an accurate clock these certificate activation & validation tests will be inaccurate. Even worse, if the time is substantially inaccurate (like a device that thinks it's 1970, for instance) then TLS handshaking will fail and the device will never connect to the Xively Service.
 
 ### Remote Updates
 
 Remote Updates is a desired feature for many reasons, but when it comes to security it's extraordinarily important. Cryptographic algorithms can be exploited over time (making their use risky), protocol implementations could be found to be faulty, and Root CA Certificates (which are stored on the device might) will expire and need to be updated. Without the ability to update the device remotely there is a high risk that the Server and Client will be running different security mechanisms, causing TLS Handshaking to Fail.
 
-The Xively Professional Services Team (XPS) can work with you to develop a firmware update strategy if you currently do not have one.
+The Xively Team (Xively-Sales@logmein.com) can work with you to develop a firmware update strategy for your platform.  We currently have a reference implementation for embedded devices in-house and will make this public by mid 2017. 
 
 # TLS Implementation Requirements
 
-The Xively Client has been tested against WolfSSL's CyaSSL library v3.2.0 and mbedTLS (formerly PolarSSL https://tls.mbed.org). Xively Professional Services (XPS) can work with your Client Application team and with WolfSSL to port that version of the CyaSSL implementation to your platform as part of the standard porting process. If your platform software or hardware has a different TLS implementation already provided, then Xively Professional Services can work with your Client Application team to use your TLS version.
+The Xively Client has been tested against [WolfSSL's library v3.2.0](https://www.wolfssl.com) and [mbedTLS](https://tls.mbed.org) (formerly PolarSSL ). If your platform e has a different TLS implementation then the following are guidelines for ensuring that your TLS implementaiton meets the same standards. Please see our [Xively C Client Porting Guide](https://github.com/xively/xively-client-c/blob/master/doc/porting_guide.md) for building the Xively C Client's Abstracted TLS BSP to support other TLS implementations.
```

Comment:
remove or add words or rephrase :)
"then the following are guidelines for ensuring"